portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,24 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import com.google.gson.Gson;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+  private ArrayList<String> json = new ArrayList<String>();
+  
+  @Override
+  public void init(){
+    json.add("Surprise!");
```

Comment:
Filling the `json` array in the init() method right now feels a bit of an overkill.
Consider instead assigning directly to the `json` member variable on line 28 above using a helper like [`Arrays.asList`](https://docs.oracle.com/javase/8/docs/api/java/util/Arrays.html#asList-T...-).

Reply:
Definition changed in commits b36b89e and 3bb0c43